src/core/Browser.js

```diff
@@ -43,10 +43,10 @@ const touch = touchNative || pointer;
 const retina = typeof window === 'undefined' || typeof window.devicePixelRatio === 'undefined' ? false : window.devicePixelRatio > 1;
 
 // @property mac: Boolean; `true` when the browser is running in a Mac platform
-const mac = typeof navigator === 'undefined' || typeof navigator.platform === 'undefined' ? false : navigator.platform.startsWith('Mac');
+const mac = typeof navigator === 'undefined' || typeof navigator?.userAgentData.platform === 'undefined' ? false : navigator.userAgent.indexOf('Mac') !== -1;
 
 // @property mac: Boolean; `true` when the browser is running in a Linux platform
-const linux = typeof navigator === 'undefined' || typeof navigator.platform === 'undefined' ? false : navigator.platform.startsWith('Linux');
+const linux = typeof navigator === 'undefined' || typeof navigator?.userAgentData.platform === 'undefined' ? false : navigator.userAgent.indexOf('Linux') !== -1;

```

Comment:
Why testing if `navigator?.userAgentData.platform` is undefined when later `navigator.userAgent` is used?